dataset/oloris-location.md

```diff
@@ -0,0 +1,96 @@
+---
+title: OpenLORIS-Location
+layout: info
+Edit: 2020-11-15
+toc: false
+commentable: false
+protected: true
+mathjax: true
+---
+
+
+
+# OpenLORIS-Location Dataset
+
+>(L)ifel(O)ng (R)obotic V(IS)ion (**OpenLORIS**) - Location Dataset (**OpenLORIS-Location**) is designed for accelerating the localization research for indoor localization, which contains variant illumination and viewpoints, severe dynamic elements, repetitive features and daily changed items.
```

Comment:
How about "... is designed for the research of indoor visual localization. It contains ..."